mesonbuild/modules/pkgconfig.py

```diff
@@ -418,14 +436,15 @@ def parse_variable_list(stringlist):
                 if any(c.isspace() for c in name):
                     raise mesonlib.MesonException('Invalid whitespace in assignment "{}"'.format(var))
 
-                if name in reserved:
+                if not dataonly and name in reserved:
```

Comment:
Hmm, why this change?

Reply:
Yeah, I suppose this is not required anymore.